### DISCUSSION TITLE
Opt out event reported as $opt_in

### DISCUSSION THREAD

**Discussion by argenkiwi** [mixpanel/docs-legacy#540] [Q&A] [answered]
I am new to the Mixpanel SDK for Android. I added some logic to allow users to opt in and out of event tracking. However, when I enable logging for Mixpanel and I make a call to the `optOutTracking()` method the logs show the following: 
```
{
  "event": "$opt_in",
  "properties": {
    ...
  },
  "$mp_metadata": {
    ...
  }
}
```
Should the event key be `$opt_out`? Is it worth raising an issue for this?

**Comment by enague** [comment, marked as answer]
Hmm, I'm wondering if you are using the optInTracking function in your code shown [here](https://mixpanel.github.io/mixpanel-android/com/mixpanel/android/mpmetrics/MixpanelAPI.html#optInTracking()). By default, I believe all users are optin until they are opted out, I would probably raise an issue on the [github repo here](https://github.com/mixpanel/mixpanel-android/issues) if you are opting a user out but you still see there data and the event $opt_in is being sent.

I would try to look at your code and see if you are calling any other methods to opt in or opt out users.
